### PR TITLE
Router and auth

### PR DESCRIPTION
**Pull request by varun-Tad**
Please Review the following files/folders.

1.context folder
2. utils folder
3. Cart Details.js

Preview : https://superlative-blini-cee770.netlify.app

### CODE REVIEW

**Comment by yaswanthmyneni** [comment]
The code is well written, remove comments, you can merge the pr.

File: dressup/src/backend/db/products.js [hunk 1]
```diff
@@ -1,30 +1,166 @@
...
+import img3 from "../../images/top.jpeg";
+import img4 from "../../images/jeans-two.jpeg";
+import img5 from "../../images/women-watch.jpeg";
+import img6 from "../../images/mens-watchOne.jpeg";
+import img7 from "../../images/MensWatch.webp";
+import img8 from "../../images/MensShirtTwo.webp";
+import img9 from "../../images/MensShirtThree.webp";
+
 export const products = [
+  // {
```
Comment (keyurshah339) [inline dressup/src/backend/db/products.js]: remove commented code from codebase

File: dressup/src/pages/CartPage/reducerFnOne.js [hunk 2]
```diff
@@ -1,54 +1,79 @@
...
-  }
-
-  state[objIndex].newPrice = item.cost * state[objIndex].quantity;
+      const totalPrice = newCartArr.reduce((acc, item) => {
+        return acc + item.quantity * item.price.discountPrice;
+      }, 0);
 
-  return state;
-};
+      console.log("totalPrice", totalPrice, typeof totalPrice);
```
Comment (keyurshah339) [inline dressup/src/pages/CartPage/reducerFnOne.js]: remove console logs

**Review by Mohitk391 — APPROVED** [review]
The code is well written and UI looks awesome, apart from suggestions from yashwant and Keyur, you can merge the branch

**Pull request merged**